core/src/main/scripts/importer/validateData.py

```diff
@@ -450,6 +468,9 @@ def _validate_file(self):
                     self.logger.warning('Ignoring invalid column header. '
                         'Continuing with validation...')
 
+            # keep track of columns that should contain unique data
+            # init a dectionary for bookkeeping
```

Comment:
I don't understand what this comment is about. Is it something you didn't intend to include in the PR?

Reply:
Well spotted. These redundant lines were removed.

Reply:
I still see them.